featuretools/tests/synthesis/test_dfs_method.py

```diff
@@ -321,6 +321,8 @@ def test_does_not_warn_with_stacking_feature(pd_es):
 
 
 def test_warns_with_unused_where_primitives(es):
+    if ks and any(isinstance(e.df, ks.DataFrame) for e in es.entities):
```

Comment:
Can you add an issue for investigating the extra warning(s)

Reply:
Created #1352 for this.